tools/all_questions.xml

```diff
@@ -2439,8 +2468,8 @@ Ebenso wie bei den Formaten gilt hier: je standardisierter, offener und etablier
 		<questionset dc:uri="https://rdmorganiser.github.io/terms/questions/ua_ruhr/metadata-and-referencing/metadata-dataset"/>
 		<is_collection>False</is_collection>
 		<order>4</order>
-		<help lang="en"/>
-		<help lang="de"/>
+    <help lang="en">Which metadata is automatically collected by computer or software, but must be verified?</help>
+    <help lang="de">Welche Metadaten werden also durch den Computer / die Software automatisch erhoben, müssen aber überprüft werden?</help>
```

Comment:
"also" entfernen

Reply:
Der Hifetext kommt von  upstream https://github.com/rdmorganiser/rdmo-catalog/commit/7786db088128fe51f84c280a4e7043a0d44470da

Sprachlich passt das 'also' nicht. Nehme es in `tools/all_questions.xml`  und  `rdmorganiser/questions/rdmo.xml` raus